Relative path in type import

Hi! Is it a known issue that mappersmith 2.39.0 that was [released yesterday](https://github.com/tulios/mappersmith/releases/tag/2.39.0) doesn’t work?

I’m getting this when doing yarn build:
```
node_modules/mappersmith/middleware/basic-auth.d.ts:1:22 - error TS2307: Cannot find module 'types' or its corresponding type declarations.

1 import { Auth } from 'types';
```

However it works with mappersmith 2.38.1.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,10 @@
 # Changelog
 
+## UNRELEASED
+
+Fixed:
+  - `mappersmith`: Typo in basic auth import no longer causes build errors (#313)
+
 ## 2.39.0
 
 Added:

--- a/src/middleware/basic-auth.ts
+++ b/src/middleware/basic-auth.ts
@@ -1,5 +1,5 @@
-import { Auth } from 'types'
-import { Middleware } from './index'
+import type { Auth } from '../types'
+import type { Middleware } from './index'
 import { assign } from '../utils'
 
 /**

--- a/src/middleware/csrf.ts
+++ b/src/middleware/csrf.ts
@@ -1,4 +1,4 @@
-import { Middleware } from './index'
+import type { Middleware } from './index'
 
 /**
  * Sets a request header with the value of a cookie from document.cookie, if it exists

--- a/src/middleware/duration.ts
+++ b/src/middleware/duration.ts
@@ -1,4 +1,4 @@
-import { Middleware } from './index'
+import type { Middleware } from './index'
 
 /**
  * Adds started_at, ended_at and duration headers to the response

--- a/src/middleware/encode-json.ts
+++ b/src/middleware/encode-json.ts
@@ -1,4 +1,4 @@
-import { Middleware } from './index'
+import type { Middleware } from './index'
 
 const mediaType = 'application/json'
 const charset = 'charset=utf-8'

--- a/src/middleware/global-error-handler.ts
+++ b/src/middleware/global-error-handler.ts
@@ -1,5 +1,5 @@
-import { Response } from '../response'
-import { Middleware } from './index'
+import type { Response } from '../response'
+import type { Middleware } from './index'
 import { configs } from '../index'
 
 export type ErrorHandlerMiddlewareCallback = (response: Response) => boolean

--- a/src/middleware/index.ts
+++ b/src/middleware/index.ts
@@ -1,5 +1,5 @@
-import { Request } from '../request'
-import { Response } from '../response'
+import type { Request } from '../request'
+import type { Response } from '../response'
 
 export type Context = object
 

--- a/src/middleware/log.ts
+++ b/src/middleware/log.ts
@@ -1,6 +1,6 @@
 import { Response } from '../response'
-import { Request } from '../request'
-import { Middleware } from './index'
+import type { Request } from '../request'
+import type { Middleware } from './index'
 
 export const defaultSuccessLogger = (message: string) => {
   const logger = console.info ? console.info : console.log

--- a/src/middleware/retry/v2/index.ts
+++ b/src/middleware/retry/v2/index.ts
@@ -1,8 +1,8 @@
-import { Middleware, ResponseGetter } from '../../index'
+import type { Middleware, ResponseGetter } from '../../index'
 import { configs } from '../../../index'
 import { assign } from '../../../utils'
 import { Response } from '../../../response'
-import { Request } from '../../../request'
+import type { Request } from '../../../request'
 
 export interface RetryMiddlewareOptions {
   readonly headerRetryCount: string

--- a/src/middleware/timeout.ts
+++ b/src/middleware/timeout.ts
@@ -1,4 +1,4 @@
-import { Middleware } from './index'
+import type { Middleware } from './index'
 
 export type Milliseconds = number
 

